messaging/src/main/java/io/lantern/messaging/Messaging.kt

```diff
@@ -850,6 +876,67 @@ class Messaging(
         }
     }
 
+    @Throws(IllegalArgumentException::class)
+    fun introduce(recipientIds: List<String>) {
+        if (recipientIds.size < 2) {
```

Comment:
Right now the frontend is also handling that (the button doesn't show up unless at least 2 contacts are selected)

Reply:
Great! This library includes checks to protect itself from invalid requests, but it's nice to have the UI prevent people from trying something invalid in the first place.